spec/controllers/miq_report_controller/trees_spec.rb

```diff
@@ -88,6 +88,8 @@
       end
 
       it 'renders list of Dashboards in Dashboards tree' do
```

Comment:
Not thrilled about these but had trouble seeding a widget set without valid data. I guess manually creating the default would have worked as well